x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/user/InternalUsers.java

```diff
@@ -122,6 +122,21 @@ public class InternalUsers {
         )
     );
 
+    public static final InternalUser SYNONYMS_USER = new InternalUser(
```

Comment:
Should we provide description (API documentation) similar to other users?

Reply:
👍  Done!